src/applications/gi/components/search/KeywordSearch.jsx

```diff
@@ -82,22 +65,42 @@ export class KeywordSearch extends React.Component {
           htmlFor="institution-search">
           {this.props.label}
         </label>
-        <Autosuggest
-          getSuggestionValue={this.clickedSuggestionValue}
-          highlightFirstSuggestion
-          onSuggestionsClearRequested={this.props.onClearAutocompleteSuggestions}
-          onSuggestionSelected={this.handleSuggestionSelected}
-          onSuggestionsFetchRequested={this.handleFetchSuggestion}
-          renderSuggestion={this.renderSuggestion}
-          shouldRenderSuggestions={this.shouldRenderSuggestions}
-          suggestions={suggestions}
-          inputProps={{
-            value: searchTerm,
-            onChange: this.handleChange,
-            onKeyUp: this.handleKeyUp,
-            'aria-labelledby': 'institution-search-label',
-            onBlur: this.handleInputBlur,
-          }}/>
+        <Downshift onChange={this.handleChange} onSelect={(item) => this.handleSuggestionSelected(item)}>
```

Comment:
I don't think you need `onChange` here, I think `onSelect` covers it. Plus, I think this causes some sluggishness with `handleChange` also being used on the input element below.

Reply:
k